review.md

```diff
@@ -0,0 +1,56 @@
+
+## Why do we code review?
+
+1. Code reviews are essential to improve the level of our code, and to prevent bugs. 
+
+2. A great and fast way for both the reviewer and the code owner to share ideas, learn, and improve their code writing and understanding skills.
+
+3. Allow us as reviewers to get to know parts of the code base we usually don't work on.
+
+4. Help our fellow team members to validate their code quality and push updates faster.
+
+## Tips for reviewing
+
+-   It is important to maintain a healthy, respectful, and civil environment. Feel free to ask questions, offer suggestions, or provide criticism, regardless of your position or level of expertise. However, it is crucial to avoid personal attacks and focus on reviewing the code rather than criticizing the individual who wrote it. Additionally, please refrain from using capital letters, using inappropriate language, or addressing others directly.\
+    Here are a few examples:
+
+| Good feedback                                                                                                      | Bad feedback                                 |
+|--------------------------------------------------------------------------------------------------------------------|----------------------------------------------|
+| "Maybe we can rephrase this to be more along those lines of: "                                                     | "This is a mess, change it to: "             |
+| "I don't believe this approach will fit what we're looking for, try finding a way of achieving it without using X” | "This code is no good, change it."           |
+| "Sorry, I might have miscommunicated that, what I meant was - ... "                                                | "this is not what I meant at all"            |
+| "This won't work, maybe try and look into X"                                                                       | "THIS IS WRONG."                             |
+| "Missing a test for X",                                                                                            | "What is it with you and missing tests?"     |
+| "I've seen this error repeated multiple times, have I mentioned X? try to look into it, it has some nice tools"    | "You keep repeating those types of mistakes" |
+| What is the reason you chose this implementation?                                                                  | huh? Why did you do that?                    |
+
+
+-   Don't forget that in the worst case scenario - you're wrong, and you learned something for the next time.
+
+
+- Remember that the goal isn't solely to suggest changes; it can also serve as an opportunity to ask questions. By asking questions, you can challenge the code owner and assist them in gaining a deeper understanding of their own code. When providing answers, code owners often realize the need for changes they hadn't considered before. Engaging in a discussion will benefit both you and the code owner.   
```

Comment:
```suggestion
- Remember that the goal isn't solely to suggest changes; it can also serves as an opportunity to ask questions. By asking questions, you can challenge the code owner and assist them in gaining a deeper understanding of their own code. When providing answers, code owners often realize the need for changes they hadn't considered before. Engaging in a discussion will benefit both you and the code owner.   
```

Reply:
I'm pretty sure the way I wrote it is correct, unless we remove the "can", in that case "serves" will fit

Reply:
anyway, done

Reply:
You're obviously right. not sure what I had in mind.